src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -38,7 +43,10 @@ const languages = {
 };
 
 export const generateContent: TranslationFn = content => {
+  const statesWithoutRespondentAnswers = [State.AwaitingAos, State.AosDrafted, State.AosOverdue];
+  const aosSubmitted = !statesWithoutRespondentAnswers.includes(<State>content.formState?.state);
```

Comment:
I think we  also need a joint application check here. Only sole applications have the respondent answers

Reply:
You could also just check to see if the document exists in the list